continuous-integration/styleci/configurations/general-php.md

```diff
@@ -0,0 +1,49 @@
+# StyleCI - General PHP
+
+The following is a recommended configuration for general PHP projects which may or may not use a framework. If you are
+using Laravel please use the alternative configuration [here](laravel.md). This configuration uses the `psr2` 
+preset with risky heuristics enabled. It is also tailored to suit our internal PHP & JavaScript guidelines.
+
+```yaml
+php:
+  preset: psr2
+  
+  risky: true
+
+  enabled:
+    - concat_with_spaces
+
+  disabled:
+    - short_array_syntax
```

Comment:
What is the reasoning behind disabling `short_array_syntax`?

Reply:
We do not have an internal guideline on enforcing the short array syntax and I believe in PSR-2 it doesn't specify it either so there was no need to enforce it. Could be worth having a discussion about though.

Reply:
`[]` `>` `array()`

Reply:
I feel like this is commonly used by everyone, but that's a very valid point. Maybe we need a separate issue just to confirm this?